config/fiab.json

```diff
@@ -0,0 +1,15 @@
+{
+  "agoraUrlRoot": "https://agora-fiab.dsde-dev.broadinstitute.org:20443",
+  "calhounUrlRoot": "https://terra-calhoun-dev.appspot.com",
```

Comment:
Note: calhoun doesn't run inside a fiab, and dev points to dev Sam for auth purposes, so that wouldn't work as expected.